src/main/webapp/app/config/constants/firebase.ts

```diff
@@ -312,7 +312,7 @@ export enum READABLE_FIELD {
   MUTATION_EFFECT = 'Mutation Effect',
   PATHOGENIC = 'Pathogenic',
   ONCOGENIC = 'Oncogenic',
-  EFFECT = 'Effect',
+  EFFECT = '',
```

Comment:
Could we use `Effect` here?
Use case would be in review mode:
![image](https://github.com/oncokb/oncokb-transcript/assets/59149377/a25fd9eb-aa0a-4b16-a237-ea28760d0494)

Reply:
Yeah for sure